common/service/config/log.go

```diff
@@ -21,8 +21,6 @@
 package config
```

Comment:
May be rename this file?

Reply:
Probably next PR? let's narrow down the scope because I already have too many conflicts with everyone.

Reply:
Actually renaming this file should have conflict with others. What is a better name for you? For me log.go looks fine because it's not a pkg name. Usually file name just for reference to help understand what's in the file.